Sources/BSON/JSON.swift

```diff
@@ -201,4 +201,54 @@ extension JSON {
     }
 }
 
-extension JSON: Equatable {}
+extension JSON: Equatable {
+    /// Compares two `JSON`s for equality. In the case of an array, order is ignored.
+    static func == (lhs: JSON, rhs: JSON) -> Bool {
+        // Linter throws the following warning:
```

Comment:
Not sure what to do about this. It probably has something to do with the fact that I basically have an O(n^2) nested for loop down at lines 234-240. Not sure how else to check array equality and ignore order.